web/html/index.html

```diff
@@ -15,90 +15,372 @@
 In this case however, we are loading a simple JS file, ../js/main.js which has some awareness of Desktop globals for interaction -->
 <html>
   <head>
-    <title>Plug-in API Example</title>
+    <title>Iframe Sample App</title>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="../css/iframe.css">
+    <script>
+      if(exports){
+        var ZoweZLUX_tempExports = exports;
+      }
+      var exports = {"__esModule": true};
+
+      //TODO: Devise a better way to check if pluginDef is available
+      let intervalId = setInterval(checkForStandaloneMode, 100);
+      function checkForStandaloneMode() {
```

Comment:
Why not build this into the iframe adapter? People would want to know. Need something off of ZoweZLUX.

Reply:
Two scenarios: in-framework but not in desktop, or out of framework entirely...